lib/kennel.rb

```diff
@@ -58,12 +61,19 @@ def generate
     end
 
     def plan
-      syncer.plan
+      the_plan = syncer.plan
+      PlanResults.new(
+        plan: the_plan
+      )
```

Comment:
```suggestion
      PlanResults.new(plan: syncer.plan)
```
...  but all this does is add indirection, so prefer keeping:
```
syncer.plan
```